.github/workflows/php-test-plugins.yml

```diff
@@ -76,3 +76,7 @@ jobs:
         run: npm run test-php -- -- -- --testsuite auto-sizes
       - name: Running multisite unit tests for Auto-sizes for Lazy-Loaded Images Plugin
         run: npm run test-php-multisite -- -- -- --testsuite auto-sizes
+      - name: Running single site unit tests for Speculation Rules Plugin
+        run: npm run test-php -- -- -- --testsuite speculation-rules
+      - name: Running multisite unit tests for Speculation Rules Plugin
+        run: npm run test-php-multisite -- -- -- --testsuite speculation-rules
```

Comment:
This is cumbersome so we should probably add shortcuts via scripts in `composer.json` and/or `package.json`.

There should also be a shortcut that runs the single site and multisite tests respectively for _all_ of these plugins at once, so that we don't have to add every single plugin to this GitHub workflow file.

Let's handle that in a separate PR though, not directly related to the Speculation Rules work.

Reply:
I opened #1017 for that.